cnf-tests/testsuites/e2esuite/test_suite_test.go

```diff
@@ -91,6 +90,21 @@ func TestTest(t *testing.T) {
 		}
 	}
 
+	RegisterFailHandler(
+		func(message string, callerSkip ...int) {
+			By("Fail Handler")
+
+			if ginkgo.CurrentSpecReport().Failed() == false {
```

Comment:
You have to remove this `if` statement: If the FailHandler is called, we don't need to check if the test is failing.

Also, `CurrentSpecReport().Failed()` is probably managed by the ginkgo.Fail()